README.md

```diff
@@ -43,31 +43,25 @@ In the upper right corner, you can switch between textbook mode, presentation mo
 
 ## What is Research Data Management?
 
-In early 2020, the COVID-19 disease, caused by the coronavirus SARS-CoV2, broke out globally, which led to the closure of many shops and businesses for quarantine reasons. The result, especially in the USA, was a large number of unemployed people who urgently needed money for their next rent payment, food or other expenses. As a consequence, the government set up a relief package for anyone who registers as unemployed - but the money didn't get to the people. This was a huge problem!
+We want to show two examples to showcase where good research data management can support you in your research.
```

Comment:
At this point, I would say these examples show why rdm is useful and important to avoid problems and desasters rather than support research.

Reply:
I think especially the first point also shows where RDM can support. Labeling data carefully is from my point of view also some kind of support.

Reply:
Okay, compromise: add "and prevent problems" to the sentence?

Reply:
Okay :) I changed it accordingly